romac4j/src/main/java/com/rakuten/rit/roma/romac4j/connection/SocketPoolFactory.java

```diff
@@ -8,16 +8,18 @@
 public class SocketPoolFactory implements PoolableObjectFactory<Connection> {
     private String nodeId;
     private int bufferSize;
+    private int timeout;
 
-    public SocketPoolFactory(String nid, int bufferSize) {
+    public SocketPoolFactory(String nid, int bufferSize, int timeout) {

```

Comment:
This change looks breaking compatibility. Could you 

1. give default value
2. Or replace this change with adding new constructor `SocketPoolFactory(String nid, int bufferSize, int timeout)`